src/profile.c

```diff
@@ -239,6 +239,49 @@ static void CheckLeaveFunctionsAfterLongjmp(void)
     }
 }
 
+static Obj JsonEscapeString(Obj param)
+{
+    Int lenString = LEN_LIST(param);
+
+    // Massively over-long string
```

Comment:
Maybe more helpful to actually spell out what's going on:
```suggestion
    // Allocate an output string with twice the size of the input
    // string, as in the worst case every single has to be escaped.
```